Web/Edubase.Web.UI/Areas/Groups/Views/Group/EditDetails.cshtml

```diff
@@ -190,7 +194,7 @@
                     }
 
                     @* OPEN DATE *@
-                    @if (Model.GroupType.OneOfThese(L.MultiacademyTrust, L.SingleacademyTrust))
+                    @if (Model.GroupType.OneOfThese(L.MultiacademyTrust, L.SingleacademyTrust, L.SecureSingleAcademyTrust))
                     {
```

Comment:
Will this enable YCS to edit open date? Spreadsheet says they're not allowed to.